KarhooUISDKTests/Mocks/KarhooSDK/MockQuote.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  MockQuote.swift
+//  KarhooUISDKTests
+//
+//  Created by Aleksander Wedrychowski on 25/10/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import KarhooSDK
+
+extension Quote {
+    static func mock(quoteType: QuoteType = .estimated, vehicleType: String = "standard") -> Quote {
+        Quote(
+            id: UUID().uuidString,
```

Comment:
There should be a helper method to get a random ID somewhere in the code. Can we reuse that?